src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
+    },
+    {
+        type: 'pattern',
+        id: 'transition-delay',
+        name: 'Transition delay',
+        prefix: '.Trsde-',
+        properties: ['transition-delay'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'i', values: ['initial']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-duration',
+        name: 'Transition duration',
+        prefix: '.Trsdu-',
+        properties: ['transition-duration'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-property',
+        name: 'Transition property',
+        prefix: '.Trsp-',
+        properties: ['transition-property'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'a', values: ['all']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-timing-function',
+        name: 'Transition timing function',
+        prefix: '.Trstf-',
+        properties: ['transition-timing-function'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
```

Comment:
Yeah, I guess we can remove it then. I thought of it as a mean for people to use these values `as-is` in case they didn't remember/don't know the abbreviation for the said values, but it is true that we do not do that for other properties, so I'll switch to false then..